scoresheets/Restaurant.tex

```diff
@@ -6,21 +6,21 @@
 
 \small\begin{scorelist}
 	\scoreheading{Regular Rewards}	
-	\scoreitem[2]{100}{Detect calling or waving customer (per order).}
-	\scoreitem[2]{100}{Reach a customer's table without prior guidance/training (per order).}
+	\scoreitem[2]{200}{Detect calling or waving customer and reach their table.}
```

Comment:
Because as a referee I cannot verify whether the robot actually detected someone who wants to order without the robot reaching the table

Reply:
Oh now I see your explanation.

I don't agree with it. As the robot could show a picture of the guest or similar. So I would go for separate items, but add the referee should be able to verify it in case of not reaching the guest close enough.

Reply:
Hm, maybe. It would need to be clear in that case. Like a cropped picture of just the guest or a bounding box.